azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -1364,6 +1386,52 @@ public String handle(ResultSet rs) throws SQLException {
     }
   }
 
+  private static class FetchRecentlyFinishedFlows implements
```

Comment:
Another strategy is to break up a larger pull request into multiple smaller ones.  Create an issue to tie them together. This can speed up the review process. In case there is a need to rollback, you have an option to roll back a portion of the changes.

e.g. adding a new method like this can go in first independently. Since it is not used yet, it is safe.